docs/resources/aom_service_discovery_rule.md

```diff
@@ -0,0 +1,148 @@
+---
+subcategory: "Application Operations Management (AOM)"
+---
+
+# huaweicloud_aom_service_discovery_rule
+
+Manages a AOM service discovery rule resource within HuaweiCloud.
+
+## Example Usage
+
+### Basic example
+
+```hcl
+resource "huaweicloud_aom_service_discovery_rule" "discovery_rule" {
+  rule_name              = "test-rule"
```

Comment:
how about just naming it `name` as it's an atrribute of rule resource.

Reply:
sure